README.md

```diff
@@ -27,23 +27,28 @@ Consult the [Python SDK reference docs](https://kubeflow-pipelines.readthedocs.i
 
 Refer to the [versioning policy](./docs/release/versioning-policy.md) and [feature stages](./docs/release/feature-stages.md) documentation for more information about how we manage versions and feature stages (such as Alpha, Beta, and Stable).
 
+## Development and Contribution
+
+For guideline on developing and contributing on Kubeflow Pipelines, please following [How to Contribute](./CONTRIBUTING.md) to make your change. For building and deploying Kubeflow Pipelines from source code, follow [ML Pipeline Development Guideline](./developer_guide.md)
```

Comment:
Suggested revision:

For guideline on developing and contributing on Kubeflow Pipelines, please following [How to Contribute](./CONTRIBUTING.md) to make your change. For building and deploying Kubeflow Pipelines from source code, follow [ML Pipeline Development Guideline](./developer_guide.md)

->

Before you start contributing to Kubeflow Pipelines, read the guidelines in  [How to Contribute](./CONTRIBUTING.md). To learn how to build and deploy Kubeflow Pipelines from source code, read the [developer guide](./developer_guide.md)

Reply:
Applied the changes